### PR TITLE
Initial Why/What/How

### LINKED ISSUE #1: Why/What/How
We need to initialise the readme with the basics of **What** SEO is, **Why** you might want to do it, and **How** to go about it.

- [x] Research SEO #2 
- [x] Write Why section #4
- [x] Write What section #10 
- [x] Write How section #12 
- [x] Further reading section #15 
~Create example #3~

### PR DESCRIPTION
**Pull request by ZooeyMiller**
Initialises the README.md with a basic why/what/how and some further reading links. 
 
fix #1 #4 #10 #9 #12 #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,171 @@
...
+[source](https://searchenginewatch.com/sew/study/2276184/no-1-position-in-google-gets-33-of-search-traffic-study)
+
+## What is SEO?
+
+Search engine optimisation is an effort to increase the traffic to a website by
+trying to make sure it appears as high in the search results as possible for
+searches related to certain queries.
+
+Before looking into SEO too much it is useful to have a little understanding
+of the way searches on the web work for which we really recommend [this short
```
Comment (finnhodgkin) [inline README.md]: I'd add a comma or new sentence before the recommendation

File: README.md [hunk 2]
```diff
@@ -1,2 +1,171 @@
...
+video on how google searches work](https://www.youtube.com/watch?v=BNHR6IQJGZs).
+[
+![screen shot of a youtube video with the title "How Search Works by Matt Cutts"](https://user-images.githubusercontent.com/21139983/30594806-4808aa00-9d47-11e7-8267-fd8bad7304ad.png)
+](https://www.youtube.com/watch?v=BNHR6IQJGZs).
+
+By understanding the way that searches work, we can try to make sure that our
+website has as high of a chance as possible to be high result in searches
```
Comment (finnhodgkin) [inline README.md]: 'as high of a chance as possible to be high result'

Typo

File: README.md [hunk 3]
```diff
@@ -1,2 +1,171 @@
...
+forward when it comes to visibility in search engines, but your
+ultimate consumers are your users, not search engines.
+
+>_[From google's own guide to SEO](https://static.googleusercontent.com/media/www.google.com/en//webmasters/docs/search-engine-optimization-starter-guide.pdf)_
+
+## How do I do it?
+
+A lot of SEO is making a clear site with good quality content that users will
+engage with, but there are some basic tips you can do on your own site which
+will make sure that search engines will show your site in relevant searches.
```
Comment (finnhodgkin) [inline README.md]: there are some basic tips that can help to make sure that search engines show your site in relevant searches.

**Comment by ZooeyMiller** [comment]
@finnhodgkin Changes made, please re-review :pray:

File: README.md [hunk 4]
```diff
@@ -1,2 +1,173 @@
...
+make sure your site shows
+up for the relevant queries, so that it gets optimum
+traffic, and all of the benefits that come with it.
+
+ You can make the best website in the world, but if _no one_ can find it, then it doesn't matter!
+
+And being high up in searches matters! A study indicated that the first result
+from a google search recieves 32.5% of clicks, where the second receives only
+17.6%, that's almost half the traffic, due to only being one position lower!
+[source](https://searchenginewatch.com/sew/study/2276184/no-1-position-in-google-gets-33-of-search-traffic-study)
```
Comment (iteles) [inline README.md:25]: 👍

**Review by iteles — APPROVED** [review]
This is a great start, thanks @ZooeyMiller !

**Pull request merged**